wiki/Community/Project_Loved/en.md

```diff
@@ -15,7 +15,7 @@ To submit a beatmap for Loved, please visit the [beatmap submission page](https:
 - [osu!catch submissions](https://loved.sh/submissions/fruits)
 - [osu!mania submissions](https://loved.sh/submissions/mania)
 
-Nominated beatmaps are then posted to the [Project Loved forum](https://osu.ppy.sh/community/forums/120), where community members may vote whether they want the beatmap to be Loved or not. Maps reaching a specified threshold percentage of "Yes" votes at their polls' end time will be moved to the Loved category.
+Nominated beatmaps are then posted to the [Project Loved forum](https://osu.ppy.sh/community/forums/120), where community members may vote whether they want the beatmap to be Loved or not. Maps with 85% of "Yes" votes at their polls' end time will be moved to the Loved category.
```

Comment:
probably should mention that 85% threshold is only for the osu! game mode, other modes use 80%

![image](https://user-images.githubusercontent.com/62819481/224049279-7f4d5fc3-42d4-4984-9a43-2ea5c52930a6.png)